### PR TITLE
Feature/1161 new issue count in dashboard

### PR DESCRIPTION
**Pull request by loic294**
image

### CODE REVIEW

File: ScoutSuite/frontend/src/components/ServiceStatus/style.scss [hunk 1]
```diff
@@ -12,11 +12,19 @@
     color: $success-color;
   }
 
-  &.issues {
+  &.critical {
+    color: $ncc-red;
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/ServiceStatus/style.scss:16]: I think we should add the variables for the color for each level now. https://ncc-scout.slack.com/files/UDCSGNLQM/F01M7FA2UN8/sc_2021-02-04_18h40m53s.png

**Review by SamuelMeilleur — APPROVED** [review]

File: ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const propTypes = {
 };
 
 const Summary = props => {
-  const services = props.services.sort((a, b) => b.flagged_items - a.flagged_items);
+  const services = props.services.sort((a, b) => b.issues.High - a.issues.High);
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/index.js:14]: we should also use `lodash/sortBy` here so we can sort for each security level.

**Review by SamuelMeilleur — CHANGES_REQUESTED** [review]

File: ScoutSuite/frontend/src/components/ServiceStatus/style.scss (hunk 1 above)
Comment (loic294) [reply ScoutSuite/frontend/src/components/ServiceStatus/style.scss]: Done!

File: ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/index.js (hunk 2 above)
Comment (loic294) [reply ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/index.js]: Fixed :)

image

**Review by SamuelMeilleur — APPROVED** [review]

**Pull request merged**